kpatch-build/insn/inat-tables.c

```diff
@@ -2,7 +2,7 @@
 /* Do not change this code. */
 
 /* Table: one byte opcode */
-const insn_attr_t inat_primary_table[INAT_OPCODE_TABLE_SIZE] = {
+static const insn_attr_t inat_primary_table[INAT_OPCODE_TABLE_SIZE] = {
```

Comment:
Is it possible to remove this file from your checking?  I'd rather leave it stock because it was copied from the kernel and we should try to keep it in sync without diverging.